cmd/kubeadm/app/constants/constants.go

```diff
@@ -403,9 +403,9 @@ var (
 	SupportedEtcdVersion = map[uint8]string{
 		12: "3.2.24",
 		13: "3.2.24",
-		14: "3.3.10",
-		15: "3.3.10",
-		16: "3.3.10",
+		14: DefaultEtcdVersion,
+		15: DefaultEtcdVersion,
+		16: DefaultEtcdVersion,
```

Comment:
i'd leave this as is, even if it duplicates the existing default version.
/cc @rosti

Reply:
I'm reverting this, if we bump `DefaultEtcdVersion` this would bump the etcd version for 14,15,16